build/Targets/Dependencies.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!--Generated file, do not directly edit.  Run "RepoUtil change" to regenerate-->
   <PropertyGroup>
-    <ManagedEsentVersion>1.9.2.0</ManagedEsentVersion>
+    <ManagedEsentVersion>1.9.2</ManagedEsentVersion>
```

Comment:
This is gonna break insertion. We need to include ".0" so that the version specified in ManagedEsent.nuspec in the package (```<version>1.9.2.0</version>```) matches this version exactly. 

The package is bad. I already notified the owning team to fix it and they did. If we update to the latest ManagedEsent we should be good.